bangc-ops/mlu_op.h

```diff
@@ -970,6 +1003,82 @@ mluOpGetAtomicsMode(mluOpHandle_t handle, mluOpAtomicsMode_t *atomics_mode);
  */
 typedef struct mluOpTensorStruct *mluOpTensorDescriptor_t;
 
+/*! The descriptor of Sequence Data that holds the dimensions,
+ * layout, data type, sequence length, padding fill, position, and scale.
+ * The total size of the tensor descriptor only support less than 2 Giga-elements.
+ * You need to call the ::mluOpCreateSeqDataDescriptor function to create a descriptor, and
+ * call the ::mluOpSetSeqDataDescriptor to set the sequence data information to the descriptor.
+ * If the sequence data is in fixed-point data type, call ::mluOpSetSeqDataDescriptorPositionAndScale
+ * to set the position and scale of the sequence data.
+ * At last, you need to destroy the descriptor at the end with the ::mluOpDestroySeqDataDescriptor
+ * function.
+ */
+typedef struct mluOpSeqDataStruct *mluOpSeqDataDescriptor_t;
+
+mluOpStatus_t MLUOP_WIN_API
+mluOpGetSeqDataDescriptor(const mluOpSeqDataDescriptor_t seq_data_desc,
+                          mluOpSeqDataLayout_t *layout,
+                          mluOpDataType_t *dtype,
+                          int *dimNb,
+                          int dimSize[],
+                          int *seqLengthArraySize,
+                          int seqLengthArray[],
+                          void *paddingFill);
+
+// Group:Common Interface
+// Subgroup:SeqData
+/*!
+ * @brief Retrieves the sequence data descriptor \p seq_data_desc that holds the dimensions,
+ * data type, layout, padding fill and the sequence lengths of the input sequence data.
+ *
+ * @param[in] seq_data_desc
+ *   Input. The descriptor of the sequence data. For detailed information,
```

Comment:
不需要写Input， doxygen语法用@param[in] 表示入参，全文排查一下哦
```suggestion
 *   The descriptor of the sequence data. For detailed information,
```